gufe/components/explicitmoleculecomponent.py

```diff
@@ -9,6 +9,10 @@
 # typing
 from ..custom_typing import RDKitMol
 
+# globally set RDKit to include all properties when pickling an RDKit Mol
+# see issue #322: https://github.com/OpenFreeEnergy/gufe/issues/322
```

Comment:
Can you add a pickle test please? If that's the use case we're targeting then we should have a test for it.

Reply:
Will do! Good call.